docs/user/summary.md

```diff
@@ -0,0 +1,1115 @@
+custom_field_data (DATA) => custom_field_data (CustomFieldData)
+description (DATA) => description (NotFound)
+comments (DATA) => comments (TextField)
+name (DATA) => name (CharField)
+slug (DATA) => slug (NotFound)
+asns (DATA) => asns (NotFound)
+. circuits.providernetwork => circuits.providernetwork .........................
+id (DATA) (CUSTOM) => id (UUIDField)
+created (DATA) => created (DateTimeField)
+last_updated (DATA) => last_updated (DoNotImportLastUpdated)
+custom_field_data (DATA) => custom_field_data (CustomFieldData)
+description (DATA) => description (CharField)
+comments (DATA) => comments (TextField)
+name (DATA) => name (CharField)
+provider (DATA) => provider_id (ForeignKey)
+service_id (DATA) => service_id (NotFound)
+. dcim.consoleport => dcim.consoleport .........................................
+id (DATA) (CUSTOM) => id (UUIDField)
+created (DATA) => created (DateTimeField)
+last_updated (DATA) => last_updated (DoNotImportLastUpdated)
+custom_field_data (DATA) => custom_field_data (CustomFieldData)
+device (DATA) => device_id (ForeignKeyWithAutoRelatedName)
+name (DATA) => name (CharField)
+_name (DATA) => _name (PrivateProperty)
+label (DATA) => label (CharField)
+description (DATA) => description (CharField)
+module (DATA) => module (NotFound)
+cable (DATA) => cable_id (ForeignKey)
+cable_end (DATA) => cable_end (NotFound)
+mark_connected (DATA) => mark_connected (NotFound)
+_path (DATA) => _path (PrivateProperty)
+type (DATA) => type (CharField)
+speed (DATA) => speed (NotFound)
+. dcim.powerport => dcim.powerport .............................................
+id (DATA) (CUSTOM) => id (UUIDField)
+created (DATA) => created (DateTimeField)
+last_updated (DATA) => last_updated (DoNotImportLastUpdated)
+custom_field_data (DATA) => custom_field_data (CustomFieldData)
+device (DATA) => device_id (ForeignKeyWithAutoRelatedName)
+name (DATA) => name (CharField)
+_name (DATA) => _name (PrivateProperty)
+label (DATA) => label (CharField)
+description (DATA) => description (CharField)
+module (DATA) => module (NotFound)
+cable (DATA) => cable_id (ForeignKey)
+cable_end (DATA) => cable_end (NotFound)
+mark_connected (DATA) => mark_connected (NotFound)
+_path (DATA) => _path (PrivateProperty)
+type (DATA) => type (CharField)
+maximum_draw (DATA) => maximum_draw (PositiveSmallIntegerField)
+allocated_draw (DATA) => allocated_draw (PositiveSmallIntegerField)
+. dcim.poweroutlet => dcim.poweroutlet .........................................
+id (DATA) (CUSTOM) => id (UUIDField)
+created (DATA) => created (DateTimeField)
+last_updated (DATA) => last_updated (DoNotImportLastUpdated)
+custom_field_data (DATA) => custom_field_data (CustomFieldData)
+device (DATA) => device_id (ForeignKeyWithAutoRelatedName)
+name (DATA) => name (CharField)
+_name (DATA) => _name (PrivateProperty)
+label (DATA) => label (CharField)
+description (DATA) => description (CharField)
+module (DATA) => module (NotFound)
```

Comment:
what does it mean? is this information lost?

Reply:
Yes, `NotFound` fields are skipped as described above.

https://github.com/nautobot/nautobot-app-netbox-importer/pull/110/files#diff-1a4b92f2e8bf12ae4a44e4d63c8a42caa969e6f492a4b61e1ae19348b2841921R245